internal/test_linux.go

```diff
@@ -0,0 +1,32 @@
+package internal
+
+import (
+	"flag"
+	"io"
+	"testing"
+
+	"github.com/google/go-tpm/tpm2"
+
+	"github.com/google/go-tpm-tools/simulator"
+)
+
+var tpmPath = flag.String("tpm-path", "", "Path to Linux TPM character device (i.e. /dev/tpm0 or /dev/tpmrm0). Empty value (default) will run tests against the simulator.")
```

Comment:
Nit: underscores for flag names are standard for most google-ish binaries.

Reply:
Done, here and elsewhere